frame/nfts/src/lib.rs

```diff
@@ -560,6 +560,8 @@ pub mod pallet {
 		WrongOwner,
 		/// The witness data given does not match the current state of the chain.
 		BadWitness,
+		/// The witness data should be provided.
+		WitnessRequired,
```

Comment:
Let's have the new errors at the bottom. It's more backward compatible (good habbits to get into).